person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly make sure the user can also have the possibility to change the parent permission apart from the default one for a better user experience.
You may achieve this by saying: 
```suggestion
  def initialize(age, name = 'Unknown', parent_permission = true)
```
And also in the `can_use_services?` make sure you only have
```rb
of_age? || @parent_permission
```

Reply:
> * [ ]  Kindly make sure the user can also have the possibility to change the parent permission apart from the default one for a better user experience.
>   You may achieve this by saying:
> 
> And also in the `can_use_services?` make sure you only have
> 
> ```ruby
> of_age? || @parent_permission
> ```

Hi @christianbiring1  - with the existing code the user can change the parent permission by passing a keyword argument e.g. parent_permission: false or parent_permission: true

Reply:
@christianbiring1 - additionally it seems it's also recommended by rubocop, see below:
![Screenshot 2023-04-12 at 1 35 38 PM](https://user-images.githubusercontent.com/17412822/231538744-0fdd76ea-b326-4f99-8235-321a26a57ee3.png)